### ISSUE TITLE
Add Links/Videos/Photos 

### ISSUE THREAD

**Issue by mcmarti2** [FodWikis/FodWikis#31]
Add links to videos of capture, release, other ops.  Add real-world photos or portions of ISSSH drawings to wiki pages.  Like FRAM Battery Box.  Need good h/w description and links/videos embedded – Melanie Miller would be a good mentor for this.

**Comment by olsonjaredm** [comment]
Use Imagery Online: https://io.jsc.nasa.gov/app/index.cfm

**Comment by olsonjaredm** [comment]
Some high traffic pages to focus on: 
- LEE
- RWS
- MT
- Ground
- Video
- MSS General
- SSRMS
- Joints
- SPDM

See list https://mod2.jsc.nasa.gov/wiki/ROBO/index.php?title=Special%3APopularPages

**Comment by mcmarti2** [comment]
Sorry to keep asking for more specifics but this still seems very vague to me. These pages are huge with a bunch of sub-pages so are there any specific areas that need certain pictures/links/videos? I don't feel like I am a good judge on whether something needs help or not.

**Comment by olsonjaredm** [comment]
Yes, it's a big project. For now, let's add an image list to each of the main landing pages listed above. Once we do that maybe we can start adding to the subpages.
https://mod2.jsc.nasa.gov/wiki/EVA/index.php?title=Help%3AImage_Gallery_Tutorial
{{Image list | ...

See if you can find 6 ish pictures related to each of the landing pages or their ORUs - searching IO for the landing page name will get you some results. 
For Video search: VDU, CLA, CLPA or get images of SSRMS with cameras in them
For Joints search: JEU or SSRMS/SPDM joint closeups
Nevermind MSS General and Ground for now since those are vague. Shoot me an email if you get stuck on a certain page and need something more specific to search for.
